examples/shared/mbedtlsconnection.c

```diff
@@ -0,0 +1,205 @@
+#include "dtlsconnection.h"
```

Comment:
This file should have some copyright statement.

Reply:
As soon as you tell me which copyright statement I should add I will add it.